modules/drivers/bigquery-cloud-sdk/test/metabase/driver/bigquery_cloud_sdk_test.clj

```diff
@@ -329,12 +355,16 @@
                         (format "CREATE TABLE %s (company STRING, founded DATETIME)
                                 PARTITION BY DATE(founded)
                                 OPTIONS (require_partition_filter = TRUE);"
-                                (fmt-table-name "partitioned_by_datetime"))
+                                (fmt-table-name "partition_by_datetime"))
                         (format "INSERT INTO %s (company, founded)
                                 VALUES ('Metabase', DATETIME('2014-10-10 00:00:00')),
                                 ('Tesla', DATETIME('2003-07-01 00:00:00')),
                                 ('Apple', DATETIME('1976-04-01 00:00:00'));"
-                                (fmt-table-name "partitioned_by_datetime"))
+                                (fmt-table-name "partition_by_datetime"))
+                        (format "CREATE MATERIALIZED VIEW %s AS
+                                SELECT company AS ev_company FROM %s WHERE founded = DATETIME('2003-07-01 00:00:00');"
+                                (fmt-table-name "mv_partition_by_datetime")
+                                (fmt-table-name "partition_by_datetime"))
```

Comment:
This tests the case that there's a materialized view with a range partition but doesn't test the case that there's a materialized view with a time partition. Since checking range partitions vs time partitions are defined in separate functions I think we should cover them both with tests, in case they evolve independently

Reply:
added one here 1b5f9ca770